ws/CMakeLists.txt

```diff
@@ -51,4 +53,13 @@ add_executable(ws
   ws.cpp)
 
 target_link_libraries(ws ixwebsocket)
+
```

Comment:
Does that one means that we need OpenSSL on Windows ?
Could we make that one use the TLS macro, or be Linux only ?

Reply:
we don't build we on windows currently as there are many errors.